doc/nrf/libraries/others/uart_async_adapter.rst

```diff
@@ -0,0 +1,103 @@
+.. _lib_uart_async_adapter:
+
+UART Async Adapter
```

Comment:
This one is tricky. We are using both convention in our code. I would preffer to use UART Async Adapter as it differentiates inside the documentation.
Why `UART Async adapter` - what is the reasoning for that?

Reply:
From Jamie Nøvik: "Maybe a dumb question, but is 'async' needed since that's part of UART? I'm not finding many hits on 'UART async adapter', but my initial impression is #2 (UART async adapter) looks correct as this seems like a common noun."